test/tests/functional/pbs_svr_dyn_res.py

```diff
@@ -371,24 +374,15 @@ def test_res_size(self):
         # The job shouldn't run
         a = {'job_state': 'Q', 'comment': job_comment}
         self.server.expect(JOB, a, id=jid2, attrop=PTL_AND)
-
-        # Delete jobs
         self.server.deljob(jid1, wait=True, runas=TEST_USER)
         self.server.deljob(jid2, wait=True, runas=TEST_USER)
 
         # Submit jobs again
-        a = {'Resource_List.foobar': '50gb'}
+        a = {'Resource_List.foobar': '100gb'}
         j1 = Job(TEST_USER, attrs=a)
         jid1 = self.server.submit(j1)
-
-        a = {'Resource_List.foobar': '50gb'}
-        j2 = Job(TEST_USER, attrs=a)
-        jid2 = self.server.submit(j2)
-
-        # Both jobs must run successfully
-        a = {'job_state': 'R', 'Resource_List.foobar': '50gb'}
+        a = {'job_state': 'R', 'Resource_List.foobar': '100gb'}
         self.server.expect(JOB, a, id=jid1)
-        self.server.expect(JOB, a, id=jid2)
 
     @skipOnCpuSet
```

Comment:
shall we remove this skip as well? the test case does not look like setting ncpus.

Reply:
@prakashcv13: As scope of this PR, I have just modified the tests which are mentioned in the PR. 
I see only one of the tests now use the decorator so I will change that as well and update the description of the PR accordingly.